.gitlab-ci.yml

```diff
@@ -79,7 +79,7 @@ before_script:
       - config/coq_config.ml
       - config/coq_byte_config.ml
       - config/dune.c_flags
-    expire_in: 1 month
+    expire_in: 1 week
```

Comment:
I'm concerned this may make it harder to run the minimizer, which relies on being able to download the artifacts from the base of the PR branch. See also https://github.com/coq/coq/commit/7f75ce0d972e63677e9571954c0d2f22fb1a7699

Would it be possible to expire artifacts more dynamically with a coqbot or CI job?  I'm thinking something like:
Whenever a CI pipeline completes successfully in master, the following non-bench artifacts are deleted:
  1. those of any already-merged PR
  2. those of any previous commit on master, excepting those commits which are the base commit of some PR which has not had a successful full CI run

I guess an alternative to this would be to have the bug minimizer fall back to the most recent successful pipeline of the master branch of Coq, emitting a warning when doing so.  (cc @Zimmi48 does this seem feasible?  I guess this would involve changing
https://github.com/coq/bot/blob/f17df6c5db449c51cb7405788f620489bbc3ee87/src/actions.ml#L1891-L1902 and https://github.com/coq/bot/blob/f17df6c5db449c51cb7405788f620489bbc3ee87/src/actions.ml#L2026-L2031
to also pass the base branch name to `minimize_failed_tests`, and I guess `ci_minimization_extract_job_specific_info` somehow has to find the most recent pipeline of the base branch which succeeded on the corresponding job?  I'm not sure how to structure the relevant query / get the right information, though, any ideas @SkySkimmer @Zimmi48 ?)

Reply:
This has been discussed during the Call yesterday https://github.com/coq/coq/wiki/Coq-Call-2024-11-05 . The main remark was that most minimizer run are launched rapidly after PR opening.